Makefile

```diff
@@ -166,6 +172,45 @@ bundle: manifests $(KUSTOMIZE) $(OPERATOR_SDK)
 bundle-build:
 	$(OCI_RUNTIME) build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
 
+# Create a new local kind cluster.
+KIND_CLUSTER_NAME = loki-cluster
+.PHONY: kind-create-cluster
+kind-create-cluster: $(KIND)
+	$(KIND) create cluster --name $(KIND_CLUSTER_NAME)
+
+# Destroy the local kind cluster.
+.PHONY: kind-delete-cluster
+kind-delete-cluster: $(KIND)
+	$(KIND) delete cluster --name $(KIND_CLUSTER_NAME)
+
+# Deploy the operator-lifecycle-manager
+# on local kind cluster.
+.PHONY: kind-install-olm
+kind-install-olm: kind-create-cluster
+	./hack/olm-install.sh $(OLM_VERSION)
+
+# Deploy the operator bundle and the operator via OLM
+# on local kind cluster.
+LOKI_OPERATOR_SECRET_NAME = loki-operator-metrics
+.PHONY: kind-olm-deploy
+ifneq ($(findstring $(KIND_CLUSTER_NAME),$(shell $(KIND) get clusters)),$(KIND_CLUSTER_NAME))
+kind-olm-deploy: kind-install-olm olm-prepare-deploy $(CFSSL) $(CFSSLJSON)
+else
+kind-olm-deploy: olm-prepare-deploy $(CFSSL) $(CFSSLJSON)
+endif
```

Comment:
This is going to execute `kind get clusters` every time **any** make target is executed.